.appveyor.yml

```diff
@@ -9,9 +9,6 @@ environment:
     secure: ipv/06DzgA7pzz2CIAtbPxZSsphDtF+JFyoWRnXkn3O8j7oRe3rzqj3LOoq2DZp4
 
   matrix:
-    - CONFIG: win_python2.7
-      CONDA_INSTALL_LOCN: C:\Miniconda36-x64
-
```

Comment:
Hmm...seems to now be dropping `win_python2.7`. 😕

Reply:
Can you revert the last 2 commits and fix https://github.com/conda-forge/backports.lzma-feedstock/pull/13#discussion_r196468495?

Reply:
Why? They seem to be necessary to workaround issue ( https://github.com/conda/conda-build/issues/2958 ).

Reply:
These have been reverted since `conda-build` got fixed. Thanks for solving that issue.